keras2onnx/ke2onnx/activation.py

```diff
@@ -51,29 +53,37 @@ def convert_keras_activation(scope, operator, container):
     input_name = operator.input_full_names[0]
     output_name = operator.output_full_names[0]
     activation = operator.raw_operator.activation
+    activation_type = type(activation)
     if activation in [activation_get('sigmoid'), keras.activations.sigmoid]:
         apply_sigmoid(scope, input_name, output_name, container)
     elif activation in [activation_get('tanh'), keras.activations.tanh]:
         apply_tanh(scope, input_name, output_name, container)
-    elif activation in [activation_get('relu'), keras.activations.relu]:
+    elif activation in [activation_get('relu'), keras.activations.relu] or \
+            activation_type == keras.layers.advanced_activations.ReLu:
         apply_relu(scope, input_name, output_name, container)
-    elif activation in [activation_get('softmax'), keras.activations.softmax]:
+    elif activation in [activation_get('softmax'), keras.activations.softmax] or \
+            activation_type == keras.layers.advanced_activations.Softmax:
         apply_softmax(scope, input_name, output_name, container, axis=-1)
-    elif activation in [activation_get('elu'), keras.activations.elu]:
+    elif activation in [activation_get('elu'), keras.activations.elu] or \
+            activation_type == keras.layers.advanced_activations.ELU:
         apply_elu(scope, input_name, output_name, container, alpha=1.0)
     elif activation in [activation_get('hard_sigmoid'), keras.activations.hard_sigmoid]:
         apply_hard_sigmoid(scope, input_name, output_name, container, alpha=0.2, beta=0.5)
     elif activation in [activation_get('linear'), keras.activations.linear]:
         apply_identity(scope, input_name, output_name, container)
     elif activation in [activation_get('selu'), keras.activations.selu]:
         apply_selu(scope, input_name, output_name, container, alpha=1.673263, gamma=1.050701)
-    elif activation in [relu6] or activation.__name__ == 'relu6':
+    elif activation_type == keras.layers.advanced_activations.LeakyReLU:
+        apply_leaky_relu(scope, input_name, output_name, container, alpha=activation.alpha.item(0))
+    elif activation_type == keras.layers.advanced_activations.PReLU:
+        apply_prelu(scope, input_name, output_name, container, slope=operator.raw_operator.get_weights()[0])
+    elif activation in [relu6] or (hasattr(activation, '__name__') and activation.__name__ == 'relu6'):
         # relu6(x) = min(relu(x), 6)
         np_type = TENSOR_TYPE_TO_NP_TYPE[operator.inputs[0].type.to_onnx_type().tensor_type.elem_type]
         zero_value = np.zeros(shape=(1,), dtype=np_type)
         apply_relu_6(scope, input_name, output_name, container,
                      zero_value=zero_value)
-    elif activation.__name__ in ['swish']:
+    elif hasattr(activation, '__name__') and activation.__name__ == 'swish':
```

Comment:
I think this change will break swish, since you get type of activation, i guess the __name__ will become 'function' instead of 'swish'.

Reply:
this is still `activation` not `activation_type` for `swish`.